src/Umbraco.Web.UI.Client/src/views/common/dashboard.controller.js

```diff
@@ -34,9 +35,25 @@ function DashboardController($scope, $routeParams, dashboardResource, localizati
         $scope.dashboard.tabs.forEach(function(tab) {
             tab.active = false;
         });
```

Comment:
This should also check `$scope.dashboard.tabs` before calling `forEach` on it (as it might still be undefined):
```suggestion
        if ($scope.dashboard.tabs && $scope.dashboard.tabs.length > 0) {
            $scope.dashboard.tabs.forEach(function(tab) {
                tab.active = false;
            });
        }
```